fract4dgui/main_window.py

```diff
@@ -442,6 +442,10 @@ def set_window_title(self):
 
         self.window.set_title(title)
 
+    def first_draw(self, *args):
+        self.on_fractal_change()
+        self.f.thaw()
```

Comment:
Appears that this thaw() is causing a draw and could go (which means first_draw() can go too). e.g. update_subfracts() calls freeze() and thaw().

Reply:
But, e.g. moving the fractal doesn't call freeze(), thaw() i.e. a thaw() is essential.